ClassifierService.cs

```
@@ -30,42 +42,16 @@ public async Task<List<Classification>> ClassifyText(string message)
             RequestUri = new Uri("https://api.cohere.ai/classify"),
             Headers =
             {
-                { "accept", "application/json" },
-                { "Cohere-Version", "2022-12-06" },
-                { "authorization", "Bearer "+configuration["CohereApiKey"] },
+                {"accept", "application/json"},
+                {"Cohere-Version", "2022-12-06"},
+                {"authorization", "Bearer " + configuration["CohereApiKey"]},
             },
             Content = new StringContent(json, Encoding.UTF8, "application/json")
         };
-        using (var response = await client.SendAsync(request))
-        {
-            response.EnsureSuccessStatusCode();
-            var body = await response.Content.ReadAsStringAsync();
-            var bodyResponse = JsonSerializer.Deserialize<ClassificationResponse>(body);
-            if (bodyResponse != null)
-            {
-                return bodyResponse.classifications;
-            }
-        }
-        return null;

```

This should throw an exception or an empty list, but let's just simplify it and assume it goes well. No need for this to be production code.